keepassxc-browser/content/keepassxc-browser.js

```diff
@@ -719,22 +719,26 @@ kpxc.fillInFromActiveElement = async function(passOnly = false) {
     }
 
     const el = document.activeElement;
-    if (el.nodeName !== 'INPUT') {
-        // No active input element selected -> fill the first combination found
-        if (kpxc.combinations.length > 0) {
-            kpxc.fillInCredentials(kpxc.combinations[0], kpxc.credentials[0].login, kpxc.credentials[0].uuid, passOnly);
-
-            // Focus to the input field
-            const field = passOnly ? kpxc.combinations[0].password : kpxc.combinations[0].username;
-            if (field) {
-                field.focus();
+
+    if (kpxc.combinations.length > 0 && kpxc.settings.autoCompleteUsernames) {
+        const field = passOnly ? kpxc.combinations[0].password : kpxc.combinations[0].username;
+        if (field && field.id !== el.id) {
```

Comment:
Remove this if statement, just focus on the field no matter what.

Reply:
done.

This seems to be working now but it kind of breaks the login at accounts.google.com - at least for me. It still works fine for the username, but fails on the second "page" of their login flow when you have to enter the password:

* `Alt+Shift+i` does not have any effect at all
* `Alt+Shitft+u` will display the autocomplete list, but it is placed in the upper left corner of the screen.

Reply:
Gotta check it.

Reply:
I think the reason is that on the password page `Alt+Shift+u` also tries to focus the username field and display the autocomplete list there. But the username field does not "exist" on the password page - at least not visibly.

So this might also affect other pages with this kind of two-step login, like Paypal for instance.

Reply:
It's logical because filling username **and** password is not working with only password fields. You'll have to use _Fill Password_ instead.

Reply:
Unfortunately,
> Alt+Shift+i does not have any effect at all